src/App.js

```diff
@@ -111,6 +112,14 @@ function App() {
     }
   };
 
+  const searchByMentorName = (mentorName) => {
+    if(mentorName.length >= 3 ){
+      setMentors(mentorsList.filter((mentor) => mentor.name.toLowerCase().indexOf(mentorName.toLowerCase()) !== -1));
+    } else {
+      setMentors(mentorsList);
```

Comment:
Thanks @mohamednajiub  for the contribution.

You need to link this filter with the other filters: by skill, country, hearted mentors.
Please, see the function: **filterMentors** and start from there.

Reply:
Thanks @islamCodehood  for this hint I'll fix it.